docs/src/modules/components/TreeItem2Anatomy.js

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { useTheme } from '@mui/material/styles';
+import Card from '@mui/material/Card';
+import CardMedia from '@mui/material/CardMedia';
+
+export default function TreeItem2Anatomy() {
+  const { palette } = useTheme();
+
+  const src =
+    palette.mode === 'light'
+      ? '/static/x/tree-view-illustrations/tree-item-light.png'
+      : '/static/x/tree-view-illustrations/tree-item-dark.png';
+  return (
+    <Card elevation={0} variant="outlined" sx={{ padding: 4 }}>
+      <CardMedia
```

Comment:
This image width growth too much. src is 1440px so can only support up to 720px without looking blurry (desktop width is more like 740px).